docs/guides/big-data.md

```diff
@@ -0,0 +1,141 @@
+---
+description: Learn tips for using Prefect when you have large amounts of data.
```

Comment:
```suggestion
description: Tips for using Prefect when passing large data objects between tasks.
```

Reply:
In a number of the examples, data isn't passed and in some it is just acted on, so would prefer more general phrasing. I'm open to some other phrasing. Do you think "Tips for using Prefect when dealing with big data?" would be preferable?

Reply:
Makes sense!